lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -1,4 +1,19 @@
 (function() {
+  var self = this;
+  var reporter = new window.ZendeskReporter();
+  function wrapped$() {
+    reporter.increment(
+      'app_framework.app_scope.violated',
+      1,
+      ['origin:' + self.location.hostname]
+    );
+    return self.$.apply(null, arguments);
+  }
+
+  var wrappedWindow = this.$.extend(true, {}, self, {$: wrapped$});
```

Comment:
NOTE: Will need a guard here too, Chat does not use jQuery.

Reply:
jQuery is a hard dependency of ZAF, so we include it in chat here: https://github.com/zopim/meshim-frontend/blob/master/src/meshim/dashboard/controllers/ZendeskApps.js#L153.

Reply:
Right, completely forgot about this!